File: sprint4/By languages/C++/lesson1code1.cpp
#include <iostream>
#include <map>
#include <string>

using namespace std;

int main() {
    map<string, string> countryByCity = {
        {"Moscow", "Russia"},
        {"Tula", "Russia"},
        {"London", "UK"},
        {"Paris", "France"},
        {"Leon", "France"},
        {"San Francisco", "USA"},
        {"New York", "USA"}
    };

    // Присвоить значение по ключу
    countryByCity["Saint Petersburg"] = "Russia";

    // Получить значение по ключу
    string country = countryByCity["Tula"];
    cout << "Tula is located in " << country << endl;

    return 0;
}

File: sprint4/By lessons/lesson1code1/lesson1code1.swift
var countryByCity = [
    "Moscow": "Russia",
    "Tula": "Russia",
    "London": "UK",
    "Paris": "France",
    "Leon": "France",
    "San Francisco": "USA",
    "New York": "USA"
]

// присвоить значение по ключу
countryByCity["Saint Petersburg"] = "Russia"

// получить значение по ключу
let country = countryByCity["Tula"] 
print("Tula is located in \(country)")
